.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,8 +22,8 @@
 ## For reviewers
 
 <!-- Don't remove the checklist below. -->
-- [ ] Check that the PR title is short, concise, and will make sense 1 year
```

Comment:
I would keep this checkmark, and rephrase it like:

> Make sure the PR title is appropriate for inclusion in the release notes.

Reply:
In order to highlight its importance? That sounds like a good idea but it would defeat the entire purpose of this PR if reviewers kept forgetting to check it afterwards.. How about highlighting the point with italic or bold font or some other way?

Reply:
I mean. This is the single biggest pain point in making a release. Having this be a good summary means that we can release more frequently

Reply:
oh yes it is :-)